### PR TITLE
Lps 86709 4

### PR DESCRIPTION
**Pull request by shuyangzhou**

### CODE REVIEW

**Comment by ngriffin7a** [comment]
@vicnate5 I fetched and locally built the branch associated with this PR from @shuyangzhou and tested all of the MVCBean demos. FWIW, everything passes 100%! :)

**Comment by brianchandotcom** [comment]
@ngriffin7a can you resend:

1.) Sort the constructor of by var name:

CDIBeanFilterMethod

2.) Rename beanType to beanClass everywhere because it's a Class<?>

3.) 

```
				private void _invokePortletBeanMethods(
						List<BeanPortletMethod> beanMethods,
						PortletRequest portletRequest,
						PortletResponse portletResponse,
						PortletConfig portletConfig)
					throws PortletException {

					for (BeanPortletMethod beanMethod : beanMethods) {
```

Too deep.

Make a new private method in CDIBeanPortletExtension

that takes in BeanPortletMethod as a param (and any other param you need). That way, the method in the for loop is less ugly.

4.) BeanPortletMethod beanMethod

Can you rename all these to 

BeanPortletMethod beanPortletMethod

Basically, analyze usage of all var names.. and all var names (in general) should hint at what it is.

Some examples:

User user
List<User> users
String json
JSONObject xyzJSONObject
Class<?> xyzClass

**Comment by brianchandotcom** [comment]
See https://github.com/ngriffin7a/liferay-portal/pull/83 for changes I've started to make

**Comment by ngriffin7a** [comment]
@brianchandotcom I'm on it
